docs/content/python/new-roles/getting-started.md

```diff
@@ -191,121 +203,63 @@ uses [data filtering](../../guides/data_access) to translate the
 policy's rules into a SQLAlchemy query. Only authorized objects will be
 fetched from the database.
 
-The authorized session is typically created during application
-initialization using `sqlalchemy_oso.authorized_sessionmaker`.
-Here's how we create it in GitClub:
+The authorized session is created using `sqlalchemy_oso.authorized_sessionmaker`.
```

Comment:
Also rushed.

Also, is this how we expect people to use this? I might have missed our best practice here.